src/generators/DetailsPageGenerator.ts

```diff
@@ -6,21 +6,31 @@ export const spaceToDashes = (name: string) => {
   return pathname.endsWith("-") ? pathname.substr(0, pathname.length - 1) : pathname;
 };
 
-export const generateSEOTitle = (foodName: string) => {
-  const name = foodName.replace(/\s*\([^)]*\)\s*/g, "");
-  return `Nutrition facts - ${name}`;
-};
-
-export const generateSEOMetaDescription = (foodName: string, category: string) => {
-  return `Nutrient facts from USDA and NIH with the option to filter by nutrients, age and gender for 100gm of  ${foodName} in ${category} category.`;
-};
-
 const path = require("path");
-
 const ff_nutrition_facts = require("../../data/foundation_food_nutrition_facts.json");
 const sr_legacy_nutrition_facts = require("../../data/sr_legacy_food_nutrition_facts.json");
-
 const recommendedDailyIntakes = require("../../data/rdi.json");
+interface Food {
```

Comment:
This interface is duplicate from `page.d.ts`. Consider removing this one and use from there.

Reply:
fixed